src/platforms/hubspot/actions/mappers.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  HubspotAssociationResponse,
+  HubspotCall,
+  HubspotCompany,
+  HubspotContact,
+  HubspotContactList,
+  HubspotContactListContact,
+  HubspotDeal,
+  HubspotEmail,
+  HubspotMeeting,
+  HubspotNote,
+  HubspotOwner,
+  HubspotProperty,
+  HubspotTask,
+} from '@/platforms/hubspot/schemas';
+
+export const transformOwner = (owner: HubspotOwner) => {
+  return {
+    id: owner.id,
+    createdAt: owner.createdAt,
+    updatedAt: owner.updatedAt,
+    firstName: owner.firstName,
+    lastName: owner.lastName,
+    email: owner.email,
+  };
+};
+
+export const transformContact = (contact: HubspotContact) => {
+  return {
+    id: contact.id,
+    createdAt: contact.createdAt,
+    updatedAt: contact.updatedAt,
+    firstName: contact.properties.firstname,
+    lastName: contact.properties.lastname,
+    email: contact.properties.email,
+    jobTitle: contact.properties.jobtitle,
+    phone: contact.properties.phone,
+    mobilePhone: contact.properties.mobilephone,
+    hsLeadStatus: contact.properties.hs_lead_status,
+    company: contact.properties.company,
+    hubspotOwnerId: contact.properties.hubspot_owner_id,
+  };
+};
+
+export const transformDeal = (deal: HubspotDeal) => {
+  return {
+    id: deal.id,
+    createdAt: deal.createdAt,
+    updatedAt: deal.updatedAt,
+    amount: deal.properties.amount,
+    dealName: deal.properties.dealname,
+    closeDate: deal.properties.closedate,
+    dealStage: deal.properties.dealstage,
+    hsDealStageProbability: deal.properties.hs_deal_stage_probability,
+    hsProjectedAmount: deal.properties.hs_projected_amount,
+    hsIsClosedWon: deal.properties.hs_is_closed_won,
+    hsIsClosed: deal.properties.hs_is_closed,
+    hubspotOwnerId: deal.properties.hubspot_owner_id,
+  };
+};
+
+export const transformCompany = (company: HubspotCompany) => {
+  return {
+    id: company.id,
+    createdAt: company.createdAt,
+    updatedAt: company.updatedAt,
+    name: company.properties.name,
+    website: company.properties.website,
+    industry: company.properties.industry,
+    phone: company.properties.phone,
+    address: company.properties.address,
+    city: company.properties.city,
+    state: company.properties.state,
+    zip: company.properties.zip,
+    country: company.properties.country,
+    numberOfEmployees: company.properties.numberofemployees,
+    annualRevenue: company.properties.annualrevenue,
+    description: company.properties.description,
+    hubspotOwnerId: company.properties.hubspot_owner_id,
```

Comment:
are we removing the properties subfield? no strong opinion either way but just making sure it's intentional

Reply:
Yeah, I figured that was a better DX